raw/sources.bib

```diff
@@ -2,7 +2,7 @@ @article{Lieberherr2017
     author = {Lieberherr, Ismael and Bodt, Timotheus Adrianus},
     year = {2017},
     title = {Sub-grouping Kho-Bwa based on shared core vocabulary},
-    journal = {Himalayan Linguistics}
+    journal = {Himalayan Linguistics},
     number = {16},
     volume = {2}
 }
```

Comment:
Please add the page ranage in the bibtex, once you are also on this. It is not nice if these simple things are ignored in our bibliographies, specifically given that the journal of this paper numbers pages, and given that it's already in evobib: http://bibliography.lingpy.org/raw.php?key=Lieberherr2017